.github/workflows/test.yaml

```diff
@@ -180,14 +180,38 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        feature: [alpine-3.16, alpine-3.17-openssl1.1-compat, alpine-latest]
+        feature:
```

Comment:
So 3 before with a matrix of 2 = 6
now 11 (+1 commented out) * 2 = 22 (so 16 more jobs)

Reply:
Correct. Most of these tests take at most 2 mins each